pending.py

```diff
@@ -61,17 +61,18 @@ def iterate_prs(repo, skip, users, path):
         repo = pr.head.repo
         user = pr.head.repo.owner.login if repo else "(unknown)"
         labels = list(label.name for label in pr.labels)
+        include_label = 'staging:()'.format(args.main)
```

Comment:
Small typo:

```suggestion
        include_label = 'staging:{}'.format(args.main)
```
Or maybe use an f-string but the rest of the code is still using `.format()` so it's probably better to remain consistent.  Maybe the code could be improved with more recent Python3 constructs as a follow-up.

Reply:
This is why I used `.format()`, I didn't want to break the convention. I'll place a follow-up PR once this one is merged

Reply:
OK. The `()` vs `{}` typo needs to be fixed regardless of the formatting method, just to be sure that doesn't get lost in the discussion.